wagtail_localize/static_src/editor/components/TranslationEditor/footer.tsx

```diff
@@ -103,7 +103,7 @@ const EditorFooter: FunctionComponent<EditorProps> = ({
     if (perms.canUnpublish) {
         actions.push(
             <a className="button action-secondary" href={links.unpublishUrl}>
-                <Icon name="download-alt" />
+                <Icon name="download" />
```

Comment:
Doesn't work with Wagtail < 5.0. Ditto for all icon changes.
The solution would be to add the old icons and register them for Wagtail < 5.0

Reply:
looks like I was hasty. 4.0 added all the correctly named icons, so this was good